.github/workflows/unstable-generation.yml

```diff
@@ -47,38 +47,38 @@ jobs:
         run: echo "::set-output name=count::$(git status -s | grep ts | wc -l)"
 
       - name: Get OpenApi version
-        if: ${{ steps.diff.outputs.count }} > 0
+        if: ${{ steps.diff.outputs.count > 0}}
```

Comment:
This is super minor, but the spacing is inconsistent in this file (no space before the end curly braces) ` }}`.